staking-pool/src/enumeration.rs

```diff
@@ -51,11 +51,11 @@ impl StakingContract {
     }
```

Comment:
Please also return the _user point_ and _user tier_ on _get_account_info_ function (line 49). This function is used for FE & cross contract call.